code/game/objects/items/weapons/ipc_scanner.dm

```diff
@@ -0,0 +1,33 @@
+/obj/item/ipc_tag_scanner
```

Comment:
How can that be acquired in the game? - I dont see it being added anywhere.

Reply:
Lore team wants it added to cargo, not be a roundstart thing for the crew.

Reply:
Discussed with the deputy loremaster, the outlined changes are alright with them.
Please ensure it is available via RnD and at Robotics at roundstart.